src/cli/cli.cpp

```diff
@@ -1279,6 +1279,15 @@ otError Interpreter::ProcessCsl(uint8_t aArgsLength, char *aArgs[])
             SuccessOrExit(error = ParseAsUint32(aArgs[1], timeout));
             SuccessOrExit(error = otLinkCslSetTimeout(mInstance, timeout));
         }
+#if OPENTHREAD_CONFIG_MAC_SSED_TO_SSED_LINK_ENABLE
+        else if (strcmp(aArgs[0], "center") == 0)
```

Comment:
As discussed, there should be state to be set, `center`, `end` and `disabled`. So how about,

`> csl s2s state off/center/end`

Using a subcommand `state` would allow us to add more s2s related parameters into CLI command later, for example, beacon period.

`> csl s2s beacon interval(0 to disable beacon)`